### PR TITLE
v0.6.0a9.post2 release

### PR DESCRIPTION
**Pull request by dcolligan**

### CODE REVIEW

File: docs/environment.yml [hunk 1]
```diff
@@ -13,8 +13,8 @@ dependencies:
 - wheel=0.29.0=py27_0
 - zlib=1.2.8=3
 - pip:
-    - git+git://github.com/ga4gh/ga4gh-client.git@master#egg=ga4gh_client
-    - git+git://github.com/david4096/schemas.git@fixpip#egg=ga4gh_schemas
+    - ga4gh_common==0.0.5
+    - ga4gh_schemas==0.6.0a9
```
Comment (david4096) [inline docs/environment.yml]: We should be on post2 schemas around the board I think https://pypi.python.org/pypi/ga4gh-schemas/0.6.0a9.post2

File: requirements.txt [hunk 2]
```diff
@@ -3,7 +3,7 @@
 #
 # These requirements are read directly into setup.py, so specify them
 # in the order that they should be in in setup.py
-ga4gh_common
-ga4gh_schemas
+ga4gh_common==0.0.5
+ga4gh_schemas==0.6.0a9
```
Comment (david4096) [inline requirements.txt]: Yeah, post2

**Comment by david4096** [comment]
Thank you @dcolligan ! Schemas has a post2 version, though I think the content of them may be documentation only so you're not going to see it fail any tests.

**Comment by dcolligan** [comment]
@david4096 fixed it anyways

**Pull request merged**